src/main/java/org/opentripplanner/routing/vehicle_parking/VehicleParkingService.java

```diff
@@ -30,4 +67,8 @@ public Stream<VehicleParking> getCarParks() {
   public Stream<VehicleParking> getVehicleParkings() {
     return vehicleParkings.stream();
   }
+
+  public Map<VehicleParkingGroup, Set<VehicleParking>> getVehicleParkingGroups() {
+    return vehicleParkingGroups;
```

Comment:
```suggestion
    return Map.copyOf(vehicleParkingGroups);
```

Reply:
This is no longer needed because the fields are immutable